<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lidar</class>
 <widget class="QMainWindow" name="Lidar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lidar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="7" colspan="5">
     <widget class="QCustomPlot" name="ocounter_plot" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="4" colspan="2">
     <widget class="QPushButton" name="syn1">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>SYN1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5" colspan="3">
     <widget class="QPushButton" name="info">
      <property name="text">
       <string>INFO</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="lon">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>LON</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Read:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QSpinBox" name="srr_spinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="3">
     <widget class="QComboBox" name="portName">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="3">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Write:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QPushButton" name="srr">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>SRR</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="3">
     <widget class="QTextEdit" name="read_log">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="lof">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>LOF</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QPushButton" name="nim">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>NIM</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="nim1">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>NIM1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="3">
     <widget class="QTextEdit" name="write_log">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="8">
     <widget class="QCustomPlot" name="ether_plot" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="ver">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>VER</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="rst">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>RST</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="2">
     <widget class="QPushButton" name="syn2">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>SYN2</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QPushButton" name="tcp_connect">
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="3">
     <widget class="QPushButton" name="connected">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  
    background-color:  rgb(0, 153, 0);
    color:  rgb(255, 255, 255);

}</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QSpinBox" name="nim_spinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="vlt">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>VLT</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="css">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>CSS</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="tm1">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>TM1</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3" colspan="5">
     <widget class="QPushButton" name="tcp_disconnect">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
